vg/draw/canvas.go

```diff
@@ -32,8 +32,47 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians, counterclockwise from
+	// the default drawing direction.
+	Rotation float64
+
+	// XAlign and YAlign specify the alignment of the text.
+	// Alignment and rotation are interrelated:
```

Comment:
Does package-level mean as a comment on the struct rather than the field, or does it mean adding it as part of the overall package summary comment?